ui/src/components/HookForm/index.jsx

```diff
@@ -594,6 +647,76 @@ export default class HookForm extends Component {
               </ListItem>
             </Fragment>
           )}
+          <div className={classes.inputWrapper}>
+            <List
+              className={classes.inputList}
+              subheader={
+                <ListSubheader className={classes.subheader}>
+                  Bindings
+                </ListSubheader>
+              }>
+              <ListItem>
+                <ListItemText
+                  primary={
+                    <TextField
+                      required
+                      label="Pulse Exchange"
+                      name="pulseExchange"
+                      placeholder="exchange/<username>/some-exchange-name"
+                      onChange={this.handleInputChange}
+                      fullWidth
+                      value={pulseExchange}
+                    />
+                  }
+                />
+              </ListItem>
+              <ListItem>
+                <ListItemText
+                  primary={
+                    <TextField
+                      required
+                      label="Routing Key Pattern"
+                      placeholder="#.some-interesting-key.#"
+                      name="pattern"
+                      onChange={this.handleInputChange}
+                      fullWidth
+                      value={pattern}
+                    />
+                  }
+                />
+              </ListItem>
+            </List>
+            <Tooltip title="Add Binding">
+              <IconButton
+                className={classNames(classes.iconButton, classes.plusIcon)}
+                onClick={this.handleAddBinding}>
+                <PlusIcon />
+              </IconButton>
+            </Tooltip>
+          </div>
+          <List>
+            {bindings.map(binding => (
+              <ListItem
+                className={classes.bindingListItem}
+                key={`${binding.exchange}-${binding.routingKeyPattern}`}>
+                <ListItemText
+                  disableTypography
+                  primary={
+                    <Typography variant="body2">
+                      <code>{binding.exchange}</code> with{' '}
+                      <code>{binding.pattern}</code>
+                    </Typography>
+                  }
+                />
+                <IconButton
+                  className={classes.iconButton}
+                  name={binding}
+                  onClick={() => this.handleDeleteBinding(binding)}>
+                  <DeleteIcon />
+                </IconButton>
+              </ListItem>
+            ))}
+          </List>
```

Comment:
Now that we have bindings, we should add a "Schedule" subheader below it and remove the `label="Schedule"` prop from the TextField component.

![Screen Shot 2019-05-13 at 10 09 15 AM](https://user-images.githubusercontent.com/3766511/57628063-330bbf80-7567-11e9-849e-79e4470ec68b.png)